component/src/test/java/com/celements/calendar/CalendarTest.java

```diff
@@ -249,11 +248,11 @@ public void testGetDocumentReference() {
 
   @Test
   public void testGetStartDate_newDate() {
-    java.util.Calendar gregCal = GregorianCalendar.getInstance();
+    java.util.Calendar gregCal = java.util.Calendar.getInstance();
     gregCal.add(java.util.Calendar.SECOND, -5);
     Date startDateBefore = gregCal.getTime();
     ICalendar cal = new Calendar(calDocRef, isArchiv);
-    gregCal = GregorianCalendar.getInstance();
+    gregCal = java.util.Calendar.getInstance();
```

Comment:
"Using the static factory method Calendar.getInstance() isn't a recommended approach as it will return an instance subjective to the default locale."
https://www.baeldung.com/java-gregorian-calendar
please change back to GregorianCalendar

Reply:
[GregorianCalender](https://docs.oracle.com/javase/8/docs/api/java/util/GregorianCalendar.html) doesn't implement a `getInstance()`, thus `GregorianCalendar.getInstance()` is identical to `Calendar.getInstance()` That's why our code cleanup has automatically replaced it here to avoid this misunderstanding.

Reply:
using `new GregorianCalendar` instead